python/lsst/fgcmcal/fgcmMakeLut.py

```diff
@@ -623,8 +630,8 @@ def _getThroughputDetector(self, detector, filterName, throughputLambda):
         ----------
         detector: `lsst.afw.cameraGeom._detector.Detector`
            Detector on camera
-        filterName: `str`
-           Short name for filter
+        physicalFilter: `str`
+           Physical filter name
```

Comment:
For consistency with `FilterLabel` I've been trying to replace "name" with "label" in general.